dev/reference/sql/statements/split-region.md

```diff
@@ -0,0 +1,207 @@
+
+Other splitting ways for index column are similar to that.
+
+For splitting data Regions of joint index, the only one different thing is you can specific the value of several columns.
+
+For example, index `idx3 (a, b)` contains 2 columns, a for timestamp and b for int. If you just want to split time range according to a column, you can use SQL statement for single time index, just don't specify the value of b column in `lower_value` and `upper_velue`.
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE t INDEX idx3 BETWEEN ("2010-01-01 00:00:00") AND ("2020-01-01 00:00:00") REGIONS 10;
+```
+
+If you want to do one more split according to b column at the same time, you need to specify the value of b column when spilting.
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE t INDEX idx3 BETWEEN ("2010-01-01 00:00:00", "a") AND ("2010-01-01 00:00:00", "z") REGIONS 10;
+```
+
+With the same time prefix of a column, this statement split out 10 regions according to the value of b column. If the value of speciied a column is different, you don't need to use the value of b column.
+
+#### Unevenly Split
+
+Index data can also be splited by index value specified by user.
+
+For example, there is idx4 (a,b) whose a column is varchar and b colun is timestamp.
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE t1 INDEX idx4 ("a", "2000-01-01 00:00:01"), ("b", "2019-04-17 14:26:19"), ("c", "");  
+```
+
+This statement specifies 3 value and cuts out 4 Regions. The range of each Region are shown as follows:
+
+```
+region1  [ minIndexValue               , ("a", "2000-01-01 00:00:01"))
+region2  [("a", "2000-01-01 00:00:01") , ("b", "2019-04-17 14:26:19"))
+region3  [("b", "2019-04-17 14:26:19") , ("c", "")                   )
+region4  [("c", "")                    , maxIndexValue               )
+```
+
+## pre_split_regions
+
+When using table with `shard_row_id_bits`, it is suggested to use with `pre_split_regions` if you want to spilt Region evenly at the start of building table. `pre_split_regions` spilt `2^(pre_split_regions-1)` Region evenly after the table is built.
+
+> **Note:**
+>
+> `pre_split_regions` must amount to less than or equal to `shard_row_id_bits`.
+
+### Example
+
+{{< copyable "sql" >}}
+
+```sql
+create table t (a int, b int,index idx1(a)) shard_row_id_bits = 4 pre_split_regions=3;
+```
+
+After building that table, this statement will cut out 4 + 1  Regions form table t. 4 (2^(3-1)) Regions for saving table row data,1 Region for saving idx1 index data.
+
+The range of 4 table Region are shown as follows:
+
+```
+region1:   [ -inf      ,  1<<61 )  
+region2:   [ 1<<61     ,  2<<61 )
+region3:   [ 2<<61     ,  3<<61 )
+region4:   [ 3<<61     ,  +inf  )
+```
+
+Why the amount of Region is 2^(pre_split_regions-1)? Because when using shard_row_id_bits, only positive numbers will be assigned to `_tidb_rowid`, so there is no need to do Spilt Region for the negative interval.
```

Comment:
```suggestion
> **Note:**
>
> The amount of split Region is 2^(pre_split_regions-1) because when using shard_row_id_bits, only positive numbers will be assigned to `_tidb_rowid`, so there is no need to do Spilt Region for the negative range.
```

Reply:
I think we better make this a Note.